src/kudu/consensus/leader_election.cc

```diff
@@ -145,11 +145,116 @@ void FlexibleVoteCounter::FetchTopologyInfo(
   CHECK(le_quorum_requirement);
   voter_distribution->clear();
   le_quorum_requirement->clear();
-  // TODO(ritwikyadav): Populate the out params by reading config.
+
+  std::string leader_region = "";
+
+  // Step 1: Compute number of voters in each region.
```

Comment:
this loop and the loop in GetDataCommitQuorum seems very similar.
can this not be abstracted out?

Reply:
There was only one point of difference but I have abstracted it out in a single helper function. I should have done this to begin with.